workflow/rules/predictions.smk

```diff
@@ -1,11 +1,18 @@
 
-rule write_feature_table: # write feature table to output directory for future reference
-    input:
-        feature_table_file = lambda wildcards: get_feature_table_file(wildcards.biosample)
-    output:
-        feature_table_here = os.path.join(RESULTS_DIR, "{biosample}", "feature_table.tsv")
-    run:
-        shell("cat {input.feature_table_file} > {output.feature_table_here}")
+rule make_biosample_feature_table:  # make feature table per biosample
```

Comment:
Do we still need this rule? Where is it being used? If we don't need it, we can remove creating the `ABC_BIOSAMPLES_MODELS` variable and file?

Reply:
yes, this is needed so that the genome wide feature table contains features for all the models being applied